src/com_tjvendors/admin/models/fields/foreignkey.php

```diff
@@ -14,6 +10,10 @@
 
 defined('JPATH_BASE') or die;
 
+use Joomla\CMS\Form\FormField;
+use Joomla\CMS\Factory;
+use Joomla\CMS\HTML\HTMLHelper;
+
 jimport('joomla.form.formfield');
```

Comment:
remove jimport('joomla.form.formfield'); from all field files if related class is loaded

Reply:
done